Apps/phironnet/ironnet.json

```diff
@@ -111,6 +111,55 @@
             "required": false,
             "default": 500,
             "order": 13
+        },
+        "enable_event_notifications": {
+            "description": "Ingest IronDefense Event Notifications into Phantom",
+            "data_type": "boolean",
+            "required": false,
+            "default": false,
+            "order": 4
+        },
+        "event_notif_actions": {
+            "description": "Event Notification Actions to include for ingest (if ingest is enabled).  Enter in CSV format.",
+            "data_type": "string",
+            "required": false,
+            "default": "Event Created",
+            "order": 5
+        },
+        "event_categories": {
+            "description": "Event Categories to exclude from ingest (if ingest is enabled).  Enter in CSV format.",
+            "data_type": "string",
+            "required": false,
+            "default": "",
+            "order": 6
+        },
+        "event_subcategories": {
+            "description": "Event SubCategories to exclude from ingest (if ingest is enabled). Enter in CSV format.",
+            "data_type": "string",
+            "required": false,
+            "default": "",
+            "order": 7
+        },
+        "event_severity_lower": {
+            "description": "Minimum Severity of Events to ingest (if ingest is enabled)",
+            "data_type": "numeric",
+            "required": false,
+            "default": 500,
+            "order": 8
+        },
+        "event_severity_upper": {
+            "description": "Maximum Severity of Events to ingest (if ingest is enabled)",
+            "data_type": "numeric",
+            "required": false,
+            "default": 1000,
+            "order": 9
+        },
+        "event_limit": {
```

Comment:
Lets add an option for whether events should be stored in their own container or in the alert's container.  If the user chooses to store them in the alert's container then make sure the description makes clear that event notifications prior to the event being associated with an alert will not be ingested.